app/Tquery/Config/TqConfig.php

```diff
@@ -61,6 +61,17 @@ public function addQuery(
         );
     }
 
+    /** Added in TqRequest for "distinct" queries */
+    public function addCount(): void
```

Comment:
Jak już wspominałem, nie mieszaj tego z danymi. Po to w requeście tquery mamy `columns: [{type: "column", column: columnName}]` żeby móc teraz użyć nowego type, skoro ta informacja nie pochodzi z danych. Ja bym dał `{type: "aggregate", aggregate: "count"}` albo `{type: "count"}` ewentualnie jeżeli nie przewidujemy innych agregacji. Działa dalej tak samo, czyli tworzy logiczną kolumnę count po której można sortować i filtrować jak po zwykłej kolumnie. Ale już nie ma paskudnego mieszania się że gdyby dane miały akurat kolumnę o nazwie count to w sumie nie wiadomo co się dostanie.

Jak dla mnie możesz to zahackować w backendzie jakoś byle jak, użyj sobie magicznej nazwy `"_____count"` na którą możesz przerobić to `{type: "aggregate", aggregate: "count"}` wewnętrznie jak tylko dostaniesz zapytanie. Ale nie rób mi takiego czegoś że elementy API się mieszają z danymi.

Reply:
do wyboru są teraz 3 opcje:
- tak jak jest
- zmiana kolumny na _count
- usunięcie funkcjonalności
count dodałem, bo wydawało mi się sensownym dodatkiem do grupowania

count w filtrowaniu, sortowaniu i selectowaniu zachowuje się jak kolumna, i i tak musi mieć swoją nazwę
i nie będę tworzył filtra typu aggregate tylko dlatego, że to nie jest do końca kolumna

jedyna istotna różnica jest, że nie jest dostępna dla distinct=false

Reply:
Wysłanie kolumny count w schema z nowym typem count jest OK.